node/tessel-export.js

```diff
@@ -427,6 +434,12 @@ Tessel.Pin.interruptModes = {
   low: 5,
 };
 
+Tessel.Pin.pullModes = {
+  pulldown: 0,
+  pullup: 1,
+  none: 2,
```

Comment:
Actually, on second thought I don't think this would be wise. It would require us to change the API and we would no longer be backwards compatible with the t1 firmware (because you'd have to call `pin.pull('UP')` rather than `pin.pull('pullup')` as currently documented.